examples/basic_bot_functions.py

```diff
@@ -0,0 +1,104 @@
+from pyboy.utils import WindowEvent
+import pyboy as pyboy
+import io
+
+'''
+This shows the basic functions you can use if you want to write a bot using only the game image as a human would see it.
+Uses Super Mario Land 1 as an example.
+'''
+
+path_to_rom = '../SuperMarioLand1.gb'
+
+
+def save_state_to_file(boy, path):
+	boy.save_state(open(path, "wb"))
+
+
+def load_state_from_file(boy, path, reset_keys=False):
+	boy.load_state(open(path, "rb"))
+	if reset_keys:
+		for key in range(WindowEvent.RELEASE_ARROW_UP, WindowEvent.RELEASE_BUTTON_START + 1):
```

Comment:
Arithmetic on WindowEvents is not encouraged. They are numbers at the moment, but that is not a guarantee. It is also not clear, which event you mean to use.